applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_fast_pair.conf

```diff
@@ -30,6 +36,9 @@ CONFIG_CAF_CLICK_DETECTOR=y
 
 CONFIG_CAF_LEDS=y
 
+CONFIG_CAF_BLE_SMP=y
```

Comment:
this configuration must be changed once my PR is merged

https://github.com/nrfconnect/sdk-nrf/pull/10803

Reply:
We could even switch to using your module in a separate PR if needed (the CAF implementation would not crash - only the synchronization part is missing)